lint_test/avoid_using_api/external_source/lib/external_source.dart

```diff
@@ -4,27 +4,35 @@
 class BannedCodeUsage {
   final String test4 = 'Hello World';
 
-  BannedCodeUsage();
+  BannedCodeUsage({this.parameter1, this.parameter2});
```

Comment:
Since we are adding a new functionality that is actually not related to the `external_source` functionality, we should create a separate package like we do for other cases.